ruled_labels/[DRAFT]specs_monorepo.yaml

```diff
@@ -0,0 +1,187 @@
+
+name: Monorepo labels
+description: Import on 2023-05-09
+
+codes:
+  - code: A
+    name: Action 
+    description: used with GHA and trigger a certain process
+  - code: B
+    name: Dependencies 
+    description: Upgrade dependencies
+  - code: C
+    name: Contribution 
+    description: Upgrade dependencies
+  - code: D
+    name: Auditing 
+    description: Required when there were changes in the paths [CI/CD INPUT NEW FILE PATHS]. Also used to indicate the need for a documentation audit of DevRel.
+  - code: F
+    name: Fails
+    description: Change breaks some part of the code
+  - code: I
+    name: Issues 
+    description: Categorisation of issues
+  - code: T
+    name: Topics
+    description: PR/Issue is related to this topic
+
+labels:
+  - name: A0-please_review
+    description: Pull request needs code review.
+    color: "666666"
+  - name: A1-needs_burnin
+    description: Pull request needs to be tested on a live validator node before merge. DevOps team is notified via matrix.
+    color: c7def8
+  - name: A2-insubstantial
+    description: Pull request requires no code review (e.g., a sub-repository hash update).
+    color: C6C7F8
+  - name: A3-in_progress
+    description: Pull request is in progress. No review needed at this stage.
+    color: DBC6F8
+  - name: A4-stale
+    description: Pull request did not receive any updates in a long time. No review needed at this stage. Close it.
+    color: F2C5F8
+  - name: A5-backport
+    description: Pull request is already reviewed well in another branch.
+    color: F8C5E7
+  - name: A6-unconfirmed
+    description: Issue might be valid, but it's not yet known.
+    color: "000000"
+  - name: B0-runtime_migration
+    description: PR introduces code that might require downstream chains to run a runtime upgrade.
+    color: c2e0c6
+  - name: B1-database_migration
+    description: PR introduces code that does a one-way migration of the database.
+    color: f4f5f9
+  - name: B2-node_first_update
+    description: This is a runtime change that will require all nodes to be update BEFORE the runtime upgrade.
+    color: e8d7d7
+  - name: B3-introduce_new_host_function
+    description: PR adds new host functions which is not used yet, so no need for upgrade.
+    color: f9f2f2
+  - name: B4-enable_new_host_function
+    description: PR enables new host functions which requires a node release before a runtime upgrade.
+    color: f9f2f2
+  - name: B5-enable_modified_host_function
+    description: INPUT NEEDED FROM @KOUTE
+    color: f9f2f2
+  - name: B6-disable_host_function
+    description: PR disables host functions which requires a node release before a runtime upgrade.
+    color: f9f2f2
```

Comment:
Hmm.... looking at these labels, maybe something like this to make them more unambiguous?

- `B3-add_new_host_function_to_client` - "PR adds a new host function to the client which is not yet used by the runtime"
- `B4-use_previously_unused_host_function_in_runtime` - "PR modifies the runtime to use a new host function; this raises the minimum client node version requirement for the next runtime upgrade!"
- `B5-unused_host_function_breaking_change` - "PR introduces a breaking change to the behavior of a host function which is not used by any runtime"
- `B6-remove_unused_host_function` - "PR removes a host function not used by any runtime"

Also, they all start with B, so will it be enforced that only one of these labels can be applied? I think we should make it possible to tag a PR with multiple of these tags (for example, in case someone would modify one host function and remove another in a single PR)

Reply:
currently the B* labels are forced to use one only, but with the new ones we can define which are (non) mandatory and single or multiple use. So no worries, these here won't be single use only. I will apply your suggestions to the labels